src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -110,13 +139,24 @@ export default defineComponent({
             return { 'border-left': `2px solid ${selectedColor.value}` }
         })
 
-        const rows = computed<DlTableRow[]>(() => {
-            return items.value?.map((i: DlLabelPickerItem) => ({
-                ...i,
-                name: i.displayLabel
-            }))
-        })
-
+        const mapObjects = (item: DlLabelPickerItem, callback: any) => {
```

Comment:
Please avoid using `any`, specify a type for the function.

Please sync with Venki about this mappedObjects function and types, as we just reviewed the same thing in the segmantation app